extensions/parrot-translate/src/parrot.tsx

```diff
@@ -0,0 +1,153 @@
+                                    return (
+                                        <List.Item
+                                            key={item.key}
+                                            icon={Icon.Text}
+                                            title={item.title}
+                                            subtitle={item?.subtitle}
+                                            accessoryTitle={item.phonetic}
+                                            actions={
+                                                <ListActionPanel
+                                                    queryText={inputState}
+                                                    copyMode={copyModeState}
+                                                    copyText={item?.subtitle || item.title}
+                                                    currentFromLanguage={translateFromLanguageState}
+                                                    onLanguageUpdate={updateTranslateTargetLanguage}
+                                                    currentTargetLanguage={currentTargetLanguageState}
+                                                />
+                                            }
+                                        />
+                                    )
+                                })}
+                            </List.Section>
+                        )
+                    })}
+                </Fragment>
+            )
+        }
+
+        // TODO: Try use Detail
+        return (
+            <List.Item
+                icon={Icon.XmarkCircle}
+                title={`Sorry! We have some problems..`}
+                subtitle={`code: ${fetchResultStateCode}`}
+                actions={
+                    <ActionPanel>
+                        <Action.OpenInBrowser
+                            title="Help"
+                            icon={Icon.QuestionMark}
+                            url="https://github.com/Haojen/raycast-Parrot#error-code-information"
+                        />
+                    </ActionPanel>
+                }
+            />
+        )
+    }
+    function onInputChangeEvt(queryText: string) {
+        clearTimeout(delayFetchTranslateAPITimer)
+
+        const text = queryText.trim()
+        if (text.length > 0) {
+            updateCopyModeState(() => {
+                const freshCopyModeState = detectIsUppercaseCopyOrLowerCaseCopy(text)
+
+                const freshInputValue = removeDetectCopyModeSymbol(text, freshCopyModeState)
+                updateLoadingState(freshInputValue !== inputState)
+                updateInputState(freshInputValue)
+
+                return freshCopyModeState
+            })
+
+            return
+        }
+
+        updateLoadingState(false)
+        updateTranslateResultState([])
+    }
+
+    return (
+        <List
+            isLoading={isLoadingState}
+            searchBarPlaceholder={"Translate to"}
```

Comment:
```suggestion
            searchBarPlaceholder={"Translate to"}
            throttle={true}
```

Reply:
The throttle API does not a good choice. here is why: https://github.com/raycast/extensions/issues/766